devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java

```diff
@@ -152,6 +152,8 @@ public void buildQuarkus() {
                 realProperties.setProperty(key, (String) value);
             }
         }
+        realProperties.putIfAbsent("quarkus.application.name", appArtifact.getArtifactId());
+        realProperties.putIfAbsent("quarkus.application.version", appArtifact.getVersion());
```

Comment:
BTW, these values most probably won't be meaningful. But we can review that later.